packages/core-manager/src/actions/info-core-version.ts

```diff
@@ -18,6 +18,21 @@ export class Action implements Actions.Action {
     }
 
     private async getLatestVersion(): Promise<string> {
-        return latestVersion("@arkecosystem/core");
+        return latestVersion("@arkecosystem/core", {
+            version: this.getChannel(),
+        });
+    }
+
+    private getChannel(): string {
+        const channels: string[] = ["alpha", "beta", "rc", "next"];
```

Comment:
Only `next` and `latest` are used. All of the other channels have been deprecated.